specs/core/0_beacon-chain.md

```diff
@@ -1248,7 +1248,6 @@ def state_transition(state: BeaconState, block: BeaconBlock, validate_state_root
 
 ```python
 def process_slots(state: BeaconState, slot: Slot) -> None:
-    assert state.slot < slot
```

Comment:
Removed this assertion because it broke our pattern of advancing slots and then running `apply_empty_block`.

Open to suggestions in modifying the pattern to preserve this `assert`